openscm/core/views.py

```diff
@@ -402,16 +396,24 @@ def values(self) -> _Timeseries:
             Lengths of set value and the time points number mismatch
         """
         self._read()
-        if self._timeseries is None:
-            self._timeseries = _Timeseries(self._data, self)
-        return self._timeseries
+        # if self._timeseries is None:  # @swillner I've broken the logic here somehow
```

Comment:
Uff, I think the whole system is broken here. Is having overwritable time points urgent actually?

Reply:
It makes life a lot easier in #186 so I'd like to spend a bit more time on it. I think we can do this without destroying everything, I just needed your help to see how (and think I can work it out now...)